src/components/ebay-carousel/test/test.browser.js

```diff
@@ -66,11 +70,11 @@ describe('given the carousel starts in the default state with items', () => {
             widget.on('carousel-move', moveSpy);
             widget.on('carousel-update', updateSpy);
             root.index = 1;
-            delay(done);
+            widget.subscribeTo(list).once('transitionend', done);
         });
 
         it('then it emits the marko move event', () => {
-            expect(moveSpy.calledOnce).to.equal(true);
+            // expect(moveSpy.calledOnce).to.equal(true);
```

Comment:
Assuming we're cool with killing the `move` event in favor of update, should be ok to delete all the tests like this.

Reply:
Right, will clean this up and also mention it in breaking changes.